sdcm/nemesis.py

```diff
@@ -846,6 +846,55 @@ def _add_drop_column_run_in_cycle(self):
             self._add_drop_column()
             time.sleep(1)
 
+    def choose_partitions_for_delete(self, partitions_amount, ks_cf, with_clustering_key_data=False):
+        # This parameter is maximum -partition-count that data is written with -validate-data
+        max_partitions_with_data_validate = self.cluster.params.get('max_partitions_with_data_validate', 0)
+
+        partitions_for_delete = defaultdict(list)
+        with self.tester.cql_connection_patient(self.target_node) as session:
+            session.default_consistency_level = ConsistencyLevel.QUORUM
+
+            for _ in range(partitions_amount):
+                partition_key = random.randint(max_partitions_with_data_validate,
+                                               max_partitions_with_data_validate + 2000)
+                cmd = f"paging off; select ck from {ks_cf} where partition_key={partition_key}"
+                result = self.target_node.run_cqlsh(cmd=cmd, verbose=False, target_db_node=self.target_node,
```

Comment:
Why cqlsh?

Reply:
change it to execute

Reply:
Because this query execution may take a long time. cqlsh allows to increase request timeout

Reply:
changed to use Cassandra driver